lib/opentok.js

```diff
@@ -522,6 +525,25 @@ OpenTok = function (apiKey, apiSecret, env) {
    *      target RTMP streams (or just one).
    *      </p>
    *      <p>
+   *      For HLS, include a single hls property in the outputs object. This object includes
+   *      the following optional properties:
+   *      <ul>
+   *       <li>
+   *        <code>dvr</code> (Boolean) — Whether to enable <a href="https://tokbox.com/developer/guides/broadcast/live-streaming/#dvr">DVR functionality</a> — rewinding, pausing,
+   *        and resuming — in players that support it (<code>true</code>), or not
+   *        (<code>false</code>, the default). With DVR enabled, the HLS URL will include a
+   *        <code>?DVR</code> query string appended to the end.
+   *       </li>
+   *       <li>
+   *        <code>lowLatency</code> (Boolean) — Whether to enable <a href="https://tokbox.com/developer/guides/broadcast/live-streaming/#low-latency">low-latency mode</a> for the HLS
+   *        stream. Some HLS players do not support low-latency mode. This feature is incompatible
+   *        with DVR mode HLS broadcasts.
+   *       </li>
+   *      </ul>
+   *      The HLS URL is returned in the response and in the REST method for getting information
+   *      about a live streaming broadcast.
```

Comment:
Instead of the REST method for getting information about a live streaming broadcast, this should mention the OpenTok Node SDK Broadcast object, which is returned in the methods for getting and listing broadcasts (`OpenTok.getBroadcast()` and `OpenTok.listBroadcasts()`).